clone/naverclone.css

```
@@ -0,0 +1,179 @@
+  clip: rect(0 0 0 0);
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  overflow: hidden;
+}
+
+header {
+  position: relative;
+}
+
+nav {
+  position: relative;
+}
+
+div#header-banner {
+  width: 1130px;
+  height: 160px;
+  margin: 0 auto;
+  position: relative;
+  border-bottom: 1px solid #e4e8eb;
+} 
+
+div#header-banner > h1 {
+  display:inline-block
+}
+
+div#header-banner > h1 a {
+  display: inline-block;
+  width: 222px;
+  height: 52px;
+  background-image: url(https://s.pstatic.net/static/www/img/uit/2021/sp_main_153a02.png);
+  background-size: 440px 410px;
+  background-position: 0 -158px;
+  background-repeat: no-repeat;
+  position: relative;
+  left: -8px;
+  top: 24px;
+}
+
+div#header-search {
+  display: inline-block;
+  position: relative;
+  top: 30px;
+}
+
+div#header-search > fieldset {
+  width: 586px;
+  height: 56px;
+  margin-left: 20px;
+  border: 2px solid #19ce60;
+  position: relative;
+}
+
+div#header-search > fieldset input {
+  width: 444px;
+  height: 24px;
+  padding: 20px 1px;
+  margin: 1px;
+  border: none;
+  outline: none;
+  font-size: 18px;
+}
+
+div#header-search > fieldset button {
+  width: 56px;
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  background-color: #19ce60;
+  border: none;
+}
+
+.ico_search {
+  display: inline-block;
+  width: 22px;
+  height: 22px;
+  background-image: url(https://s.pstatic.net/static/www/img/uit/2021/sp_main_153a02.png);
+  background-size: 440px 410px;
+  background-position: -204px -386px;
+  background-repeat: no-repeat;
+}
+
+.ico_weather_munji_good {
+  display: inline-block;
+  width: 18px;
+  height: 18px;
+  background-image: url(https://s.pstatic.net/static/www/img/uit/2021/sp_main_153a02.png);
+  background-size: 440px 410px;
+  background-position: -420px -132px;
+  background-repeat: no-repeat;
+  position: relative;
+  top: 3px;
+}
+
+.ico_weather_munji_soso {
+  display: inline-block;
+  width: 18px;
+  height: 18px;
+  background-image: url(https://s.pstatic.net/static/www/img/uit/2021/sp_main_153a02.png);
+  background-size: 440px 410px;
+  background-position: -420px -111px;
+  background-repeat: no-repeat;
+  position: relative;
+  top: 3px;
+}
+
+.ico_weather_munji_bad {
+  display: inline-block;
+  width: 18px;
+  height: 18px;
+  background-image: url(https://s.pstatic.net/static/www/img/uit/2021/sp_main_153a02.png);
+  background-size: 440px 410px;
+  background-position: -420px -92px;
+  background-repeat: no-repeat;
+  position: relative;
+  top: 3px;
+}
+
+div#header-movenmentsite {
+  position: absolute;
+  top: 0;
+  right: 0;
+}
+
+div#header-movenmentsite > a{
+  font-size: 11px;
+  color: black
+}
+
+div#header-nav-group {
+  width: 1130px;
+  height: 52px;
+  margin: 0 auto;
+  position: relative;
+  padding: 11px 0 11px 0;
+  border-bottom: 1px solid #e4e8eb;
+}
+
+div#header-nav-group  ul {
+  list-style: none;
+  font-size: 15px;
+  line-height: 30px;
+  font-weight: 700;
+  color: #000;
+  letter-spacing: -.3px;
```

```suggestion
  letter-spacing: -0.3px;
```

취향 차이이긴 한데 전 개인적으로 0을 붙이는게 더욱 명시적이라 좋은 것 같습니다.